docs/src/main/asciidoc/quarkus-reactive-architecture.adoc

```diff
@@ -0,0 +1,182 @@
+////
```

Comment:
The contents of this I pressume are the same as the doc you wrote and I had reviewed a week or two ago?

Reply:
yes